_posts/2017-09-25-announcing-prebid-1.0.md

```diff
@@ -0,0 +1,44 @@
+## Prebid 1.0 - here at last
+
+It's been a long time since we [first released Prebid.js](http://prebid.org/blog/happy-birthday-prebid-js); so long, in fact, that people have questioned why were are still a [pre 1.0 library.](https://github.com/prebid/Prebid.js/issues/891) While I'd like to think it's because we had everything right from the start - we definitely did not - we've been hard at work to make things even better. We've also grown a lot since then, and even [made some friends along the way](http://prebid.org/blog/announcing-prebid-org).
+
+Suffice to say, we are ready to cross over the 1.0 milestone :rocket:. A great part about us working on the Prebid 1.0 journey is that we ended up integrating many of the ideas now, instead of waiting for the big release. So, some of these features are already inside prebid.js today.
```

Comment:
Instead of "we ended up integrating many of the ideas now", would suggest "we ended up delivering many of the original ideas already."